apps/email/js/cards/compose.html

```diff
@@ -21,64 +22,70 @@
   <div data-prop="addrBar" class="cmp-envelope-bar">
     <div class="cmp-addr-bar">
       <div data-event="click:onContainerClick"
-           class="cmp-envelope-line cmp-combo">
+           class="cmp-envelope-line cmp-combo"
+           role="group" data-l10n-id="to-group">
         <span class="cmp-to-label cmp-addr-label"
-               data-l10n-id="compose-to">tO:</span>
+              data-l10n-id="compose-to" aria-hidden="true">tO:</span>
         <div class="cmp-to-container cmp-addr-container">
           <div class="cmp-bubble-container">
               <input data-prop="toNode"
                      data-event="keydown:onAddressKeydown,input:onAddressInput"
-                     dir="auto"
```

Comment:
According to the [whatwg spec](https://html.spec.whatwg.org/multipage/dom.html#the-dir-attribute), dir does not have a default value, and dir is kept for some other elements, so I would prefer to keep this attribute in unless there is other information that indicates it should be removed.